### DISCUSSION TITLE
Customize Login e Portal

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#7820] [Ideas]
First I want to congratulate you on the platform. I've been using it with good results.

Is there any prediction or possibility to customize the login and the inical screen of the welcome portal?

I'm in Brazil I would like these pages to be in Portuguese.

Same situation for the success and error messages.

**Comment by shogunpurple** [comment]
Hi @Kakovisk - yes, this is something we get a lot of requests around. There will be support coming for customisation of error/success messages and some better all round translation support. Right now there's no way to do this platform-wide, however you can make use of the custom alert action to provide alerts in other languages.

Customisation of the login page is another feature we will be implementing also - however this could be several months away.
